exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapRecordReader.java

```diff
@@ -125,8 +129,8 @@ public int next() {
 
   @Override
   public void close() throws Exception {
-//    buffer = null;
-//    in.close();
+    in.close();
+    fs.close();
```

Comment:
Since you did not open fs, I guess it's not your responsibility to close it.

Reply:
Sure, my mistake, I will remove it.